packages/cli/src/cliEntry.js

```diff
@@ -122,11 +112,6 @@ const addCommand = (command: CommandT, ctx: ContextT) => {
       opt.default,
     ),
   );
-
-  // Redefined here to appear in the `--help` section
-  cmd
-    .option('--projectRoot [string]', 'Path to the root of the project')
-    .option('--reactNativePath [string]', 'Path to React Native');
```

Comment:
Once removed, they will not appear in the `--help` for a command, right?

Reply:
I guess we can tweak `printHelpInformation` a bit to render global options too.

Reply:
Looks like they show up just right:

<img width="505" alt="Screenshot 2019-03-08 at 13 54 48" src="https://user-images.githubusercontent.com/5106466/54114025-c995f600-43e9-11e9-92ec-ca5eee927ae5.png">

Reply:
Ok, it's not right, I've added extra `.parse()` call and it skips commands. Gonna work on it

Reply:
Yeah, but this was to handle `react-native start --help` (help for a command) where they don't show up anymore.

<img width="907" alt="Screenshot 2019-03-11 at 10 41 07" src="https://user-images.githubusercontent.com/2464966/54114238-45903e00-43ea-11e9-9aa9-c33db3b97dfc.png">

Reply:
Ok, just removed this extra call and help is complete:

<img width="1238" alt="Screenshot 2019-03-11 at 10 43 30" src="https://user-images.githubusercontent.com/5106466/54114382-9869f580-43ea-11e9-92ce-adeb98e2de96.png">

Reply:
ugh

Reply:
brought it back with (imho) more appropriate comment